src/main/java/frc/robot/maps/FrancoisMap.java

```diff
@@ -52,7 +53,7 @@ public SendableSpeedController getRight() {
                 sendLeader.getEncoder().setVelocityScaleFactor(distancePerRev);
                 SendableRegistry.add(sendLeader.getEncoder(), "Right Drive");
                 SendableRegistry.enableLiveWindow(sendLeader.getEncoder());
-
+                // return sendLeader;
```

Comment:
Take us to your leader

Reply:
We are returning a Mod speed controller here do you want me just to get rid of this comment?